kayenta/kayenta-integration-tests/src/test/java/com/netflix/kayenta/configuration/TestPrometheusConfiguration.java

```diff
@@ -0,0 +1,131 @@
+package com.netflix.kayenta.configuration;
```

Comment:
Why the rename from EmbeddedPrometheusBootstrapConfiguration?  In the process, we lost the license header.

Reply:
We mainly renamed `EmbeddedPrometheusBootstrapConfiguration` to `TestPrometheusConfiguration` to clarify its intended usage. The original name included Bootstrap, which in Spring Boot contexts often implies early lifecycle loading or auto-configuration, which isn't the case now. 

This class is strictly used in test contexts to configure an embedded Prometheus instance. Renaming it with a test prefix helps prevent accidental use outside of tests and makes its role clearer to future contributors immediately.

That said, happy to revert the rename if you'd prefer we stick with the original name for consistency.

I will add the license header.

Reply:
https://github.com/spinnaker/spinnaker/pull/7053/commits/851e515d0e98a3035408fdcdc03e5dc33dd750e3

@dbyron-sf here is commit that reverted the name